domain/src/DomainValidator.php

```diff
@@ -133,7 +133,7 @@ public function checkResponse(DomainInterface $domain) {
     }
     // We cannot know which Guzzle Exception class will be returned; be generic.
     catch (RequestException $e) {
-      watchdog_exception('domain', $e);
+      //watchdog_exception('domain', $e);
```

Comment:
Do you know why this was throwing errors?

Reply:
I'm sorry I don't recall exactly at this point, but I think it was for dns lookup error. I believe I changed this because returning an exception from a validate seemed to me to be an odd thing to do when we are explicitly returning an error code as well.